try {
  throw new Error("error");
} catch (e) {
  throw e;
}
